runtime/v2/shim.go

```diff
@@ -48,6 +48,8 @@ const (
 	shutdownTimeout = "io.containerd.timeout.shim.shutdown"
 )
 
+var _ runtime.Task = &shim{}
```

Comment:
nit: should we add this closer to the `shim` struct definition? Like line 194 or something?

Reply:
done